qa/tasks/cephfs/test_client_recovery.py

```diff
@@ -383,7 +385,6 @@ def test_filelock(self):
 
         self.mount_b.check_filelock(do_flock=flockable)
 
-        # Tear down the background process
```

Comment:
removal of these comments is intentional right? Looks too trivial

Reply:
Yes. They're ultra-redundant.